src/pyodide/python-entrypoint-helper.js

```diff
@@ -190,22 +199,24 @@ async function setupPackages(pyodide) {
     pyodide.FS.writeFile(
       "/lib/python3.11/site-packages/aiohttp_fetch_patch.py",
       new Uint8Array(mod.default),
-      { canOwn: true }
+      { canOwn: true },
     );
     pyodide.pyimport("aiohttp_fetch_patch");
   }
-  if (requirements.some(req => req.startsWith("fastapi"))) {
+  if (requirements.some((req) => req.startsWith("fastapi"))) {
     const mod = await import("pyodide-internal:asgi.py");
     pyodide.FS.writeFile(
       "/lib/python3.11/site-packages/asgi.py",
       new Uint8Array(mod.default),
-      { canOwn: true }
+      { canOwn: true },
     );
   }
 
   // The main module can have a `.py` extension, strip it if it exists.
   const mainName = metadata.mainModule;
-  const mainModule = mainName.endsWith(".py") ? mainName.slice(0, -3) : mainName;
+  const mainModule = mainName.endsWith(".py")
+    ? mainName.slice(0, -3)
+    : mainName;
```

Comment:
Might be good to disable autoformat, you'll get lots of conflicts because you touch these lines in other PRs :)